modules/querier/http.go

```diff
@@ -50,6 +65,21 @@ func (q *Querier) TraceByIDHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if r.Header.Get("Tempo-query-content-type") == "application/grpc" {
```

Comment:
This block can be dropped.

Reply:
There's a standard header to use here:  "Accept".

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept

As far as the MIME type it looks like there's some discussion:

https://stackoverflow.com/questions/30505408/what-is-the-correct-protobuf-content-type

`application/protobuf` is fine